app/views/teams/show.html.erb

```diff
@@ -3,6 +3,52 @@
     <h1><%= i @team.team_type.icon %> <%= @team.identifier_name %></h1>
   </div>
   <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <div style="margin-bottom: 20px;">
```

Comment:
Use the bootstrap `mb-3` class for this kind of thing to keep things consistent across the application